aya/src/maps/mod.rs

```diff
@@ -369,14 +369,14 @@ macro_rules! impl_map_pin {
       <($($ty_param:ident),*)>
       $ty:ident
     ) => {
-            impl<T: BorrowMut<MapData>, $($ty_param: Pod),*> $ty<T, $($ty_param),*>
+            impl<T: Borrow<MapData>, $($ty_param: Pod),*> $ty<T, $($ty_param),*>
```

Comment:
The TODO at the top of the macro can go now?

Reply:
Yep!